src/main/java/io/github/railroad/settings/ui/SettingsAppearancePane.java

```diff
@@ -1,6 +1,71 @@
 package io.github.railroad.settings.ui;
 
-import javafx.scene.layout.BorderPane;
+import io.github.railroad.Railroad;
+import io.github.railroad.ui.defaults.RRVBox;
+import io.github.railroad.utility.ConfigHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.Label;
 
-public class SettingsAppearancePane extends BorderPane {
+import java.io.File;
+import java.util.Arrays;
+
+public class SettingsAppearancePane extends RRVBox {
+    private final Label title = new Label("Appearance");
+
+    private static ChoiceBox themeSelector = new ChoiceBox<String>();
+    private final Label themeOption = new Label("Select a theme:");
+
+    public SettingsAppearancePane() {
+        super();
+
+        var themeBox = new RRVBox();
+
+        title.setStyle("-fx-font-size: 20pt; -fx-font-weight: bold;");
+        title.prefWidthProperty().bind(widthProperty());
+        title.setAlignment(Pos.CENTER);
+
+        themeBox.setStyle("-fx-padding: 0 0 0 10");
+
+        var themesDir = new File("themes").listFiles();
```

Comment:
I would prefer that we avoid using java.io where possible and try to use nio. As for how you'd do that here, probably `Files.listFiles(Path)` which gives you a `Steam<Path>`.